src/test/java/seedu/address/model/person/filter/IdContainsDigitsPredicateTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.person.filter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static seedu.address.testutil.TypicalIds.ID_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIds.ID_SECOND_PERSON;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class IdContainsDigitsPredicateTest {
+
+    @Test
+    public void getIds_test_success() {
+        IdContainsDigitsPredicate predicate = new IdContainsDigitsPredicate(List.of(ID_FIRST_PERSON.value,
+                ID_SECOND_PERSON.value));
+        assertEquals(List.of(ID_FIRST_PERSON.value, ID_SECOND_PERSON.value), predicate.getIds());
+    }
```

Comment:
Seems like `getIds` is only used in this test. Maybe we can entirely remove that method.